packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -879,7 +862,7 @@ export class SingleThreadedWallet
     response.createdObjectives = storedObjectives;
 
     const channelIdsFromRequests: Bytes32[] = [];
-    const requests = (wirePayload.requests || []).map(deserializeRequest);
+    const requests = wirePayload.requests || [];
```

Comment:
Curious about this change? It doesn't seem related to the main thrust of the PR?

Reply:
Nevermind, I see that the `deserializeRequest` function is very specific to ledger proposals, and otherwise is essentially an identity operation. Good call on removing it.